<launch>
    <arg name = "csvpath" default = "$(find r6g_steppermotor)/datapoints/poses.csv" />
    <arg name = "port" default = "/dev/ttyUSB0" />
    <arg name = "delaySerial" default = "5.0" />

    <node name = "SerialSending" pkg = "r6g_steppermotor" type = "SerialSending.py" args = "$(arg port)" output = "screen" launch-prefix = "bash -c 'sleep $(arg delaySerial); $0 $@'" />
    <node name = "Detector" pkg = "r6g_steppermotor" type = "DetectorObjetos.py" args = "$(arg csvpath)" output = "screen" />
    <node name = "StepControl" pkg = "r6g_steppermotor" type = "StepControl.py" output = "screen" />
</launch>